src/o365/graph/commands/teams/teams-user-set.ts

```diff
@@ -0,0 +1,218 @@
+
+            const requestOptions: any = {
+              url: endpoint,
+              headers: Utils.getRequestHeaders({
+                authorization: `Bearer ${auth.service.accessToken}`,
+                'accept': 'application/json;odata.metadata=none'
+              }),
+            };
+
+            if (this.debug) {
+              cmd.log('Executing web request...');
+              cmd.log(requestOptions);
+              cmd.log('');
+            }
+
+            return request.delete(requestOptions);
+          }
+          else {
+            throw new Error("The specified user is already an member in the specified team, and thus cannot be demoted.");
+          }
+        }
+      })
+      .then((): void => {
+        if (this.verbose) {
+          cmd.log(vorpal.chalk.green('DONE'));
+        }
+
+        cb();
+      }, (err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
+  }
+
+  private getOwners(cmd: CommandInstance, teamId: string): Promise<void> {
+    const endpoint: string = `${auth.service.resource}/v1.0/groups/${teamId}/owners?$select=id,displayName,userPrincipalName,userType`;
+
+    return this.getAllItems(endpoint, cmd, true).then((): void => {
+      // Currently there is a bug in the Microsoft Graph that returns Owners as
+      // userType 'member'. We therefore update all returned user as owner  
+      for (var i in this.items) {
+        this.items[i].userType = 'Owner';
+      }
+    });
+  }
+
+  private getMembersAndGuests(cmd: CommandInstance, teamId: string): Promise<void> {
+    const endpoint: string = `${auth.service.resource}/v1.0/groups/${teamId}/members?$select=id,displayName,userPrincipalName,userType`;
+    return this.getAllItems(endpoint, cmd, false);
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-i, --teamId <teamId>',
+        description: 'The ID of the team for which to list users'
+      },
+      {
+        option: '-n, --userName <userName>',
+        description: 'User\'s UPN (user principal name, eg. johndoe@example.com)'
+      },
+      {
+        option: '-r, --role [type]',
+        description: 'Filter the results to only users with the given role: Owner|Member',
+        autocomplete: ['Owner', 'Member']
+      },
+
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(): CommandValidate {
+    return (args: CommandArgs): boolean | string => {
+      if (!args.options.teamId) {
+        return 'Required parameter teamId missing';
+      }
+
+      if (!Utils.isValidGuid(args.options.teamId as string)) {
+        return `${args.options.teamId} is not a valid GUID`;
+      }
+
+      if (!args.options.userName) {
+        return 'Required parameter userName missing';
+      }
+
+      if (!args.options.role) {
+        return 'Required parameter role missing';
+      } else {
```

Comment:
Since the `if` clause returns a value, there is no need to wrap the following code in an `else` block.